alembic/versions/9bea906e3151_add_bracket_id_to_user_tournament.py

```diff
@@ -0,0 +1,30 @@
+"""add bracket_id to user_tournament
+
+Revision ID: 9bea906e3151
+Revises: 35af944ce3dd
+Create Date: 2022-01-11 08:19:06.841717
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '9bea906e3151'
+down_revision = '35af944ce3dd'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('user_tournaments', sa.Column('bracket_id', sa.Integer(), nullable=True))
```

Comment:
Ohhhhh, I see. Ignore my previous comment.

Reply:
Yeah I fucked up on that first migration, figured one little extra migration file wouldn't be the end of the world, but it does mean my incompetence will be forever immortalized in the alembic history